ws-server/src/server.rs

```diff
@@ -95,7 +95,8 @@ impl Server {
 
 		let mut incoming = TcpListenerStream::new(self.listener).fuse();
 		let methods = self.methods;
-		let connections = Arc::new(RwLock::new(HashMap::new()));
+		let conn_counter = Arc::new(());
+		let is_conn = Arc::new(AtomicBool::new(true));
```

Comment:
How about calling this `shutdown`? Without the `is_` or mixing in `conn` that could mean "connections", "connected", "connecting" etc?

Reply:
fair enough, will change.